config/initializers/devise.rb

```diff
@@ -134,7 +134,7 @@
   # without confirming their account.
   # Default is 0.days, meaning the user cannot access the website without
   # confirming their account.
-  # config.allow_unconfirmed_access_for = 2.days
+  config.allow_unconfirmed_access_for = 2.days
```

Comment:
2 days is the default here, but maybe it should be more like a week?

Reply:
Since we're explicitly disallowing the thing we don't want people to do without confirming, I wonder if we should set it to [`nil` so it never kicks them out?](https://github.com/heartcombo/devise/pull/2278)

Reply:
I agree that if we're going to go with the model of preventing actions within the account based on if an email is confirmed, it's OK to always let folks log in.